endercore/src/main/java/com/enderio/core/common/util/CallOnly.java

```diff
@@ -0,0 +1,16 @@
+package com.enderio.core.common.util;
+
+import net.minecraftforge.fml.LogicalSide;
+import net.minecraftforge.fml.util.thread.EffectiveSide;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Target;
+
+/**
+ * Marks a method to only be called on a side without using the {@link net.minecraftforge.api.distmarker.OnlyIn} Annotation
```

Comment:
Why not use the onlyIn annotation?

Reply:
Because it has a big javadoc warning to not use it as it has a meaning, that the code is stripped on the other side. I don't want to strip the code and we don't build a client+server jar anyway